src/main/java/com/google/api/codegen/ApiaryConfig.java

```diff
@@ -141,6 +151,22 @@
     return mediaUpload;
   }
 
+  public String getServiceCanonicalName() {
+    return serviceCanonicalName;
+  }
+
+  public void setServiceCanonicalName(String serviceCanonicalName) {
+    this.serviceCanonicalName = serviceCanonicalName;
+  }
```

Comment:
There weren't any set calls before your change here, why did you have to add them?